src/database/validators/accounts.py

```
@@ -0,0 +1,27 @@
+        raise ValueError("Password must contain at least one digit.")
+    if not re.search(r'[@$!%*?&#]', password):
+        raise ValueError("Password must contain at least one special character: @, $, !, %, *, ?, #, &.")
+    return password
+
+
+def validate_email(user_email: str) -> str:
+    try:
+        email_info = email_validator.validate_email(user_email, check_deliverability=False)
+        email = email_info.normalized
+    except email_validator.EmailNotValidError as error:
+        raise ValueError(str(error))
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)